robot1:
  setpoint_topic: "/hydra_controller/rob1/command"
  control_freq: 250
  ros_queue_size: 1

robot2:
  setpoint_topic: "/hydra_controller/rob2/command"
  control_freq: 250
  ros_queue_size: 1

robot3:
  setpoint_topic: "/hydra_controller/rob3/command"
  control_freq: 250
  ros_queue_size: 1